cdk/App.ts

```diff
@@ -34,7 +34,6 @@ const apiStack = new ApiStack(app, 'ApiStack', {
   env: deploymentEnv
 });
 
-const apiStack = new ApiStack(app, 'ApiStack', {})
 
 networkingStack.addDependency(storageStack);
```

Comment:
Nit: we should revisit the networking stack to reduce cost. Can do a bunch of things to force things to stay within the VPC which will reduce costs. Might be able to do without an internet gateway for example